src/main/java/org/codehaus/mojo/license/AbstractAddThirdPartyMojo.java

```diff
@@ -116,6 +116,9 @@
 
     /**
      * A filter to exclude some scopes.
+     * <p>
+     * Multiple scopes are delimited by comma, eg. {@code <excludedScopes>test,provided</excludedScopes>}
+     * or {@code -Dlicense.excludedScopes="test,provided"}.
```

Comment:
`-Dlicense.excludedScopes="test,provided"` perhaps works, but the quotes are not necessary as long as there is no whitespace there. It works without quotes on all platforms. So please remove the quotes.